### ISSUE TITLE
Crash on Level End

### ISSUE THREAD

**Issue by aryareal1** [ZiLko/platformer_ghosts#12]
The latest update caused a crash on level end/finish. It happend 
before the level screen says "Level Complete".

Mod Version: 1.2.0
Platform: Android 12
Geode Version: 1.4.2
```
2024-12-06T14:42:40.757347204 [W/.geode.launche]: Cleared Reference was only reachable from finalizer (only reported once)
2024-12-06T14:42:40.777109742 [W/System]: A resource failed to call close. 
2024-12-06T14:42:49.487301358 [W/AudioTrack]: restartIfDisabled(30381): releaseBuffer() track 0xb40000708b360640 disabled due to previous underrun, restarting
2024-12-06T14:42:49.548063281 [E/Geode]: 14:42:49 ERROR [Thread #1822] [Geode]: Geode crashed! Crash dump saved to /storage/emulated/0/Android/media/com.geode.launcher/game/geode/crashlogs/Crash 2024-12-06 14.42.31.dmp
2024-12-06T14:42:51.866512205 [I/.geode.launche]: Using CollectorTypeCC GC.
2024-12-06T14:42:51.878791820 [E/.geode.launche]: Not starting debugger since process cannot load the jdwp agent.
2024-12-06T14:42:51.958163205 [I/LoadedApk]: No resource references to update in package com.transsion.theme.icon
2024-12-06T14:42:51.989514974 [I/libMEOW_gift]: open /data/performance/gift/arc.ini
2024-12-06T14:42:51.989619205 [E/QT]: [QT]file does not exist
2024-12-06T14:42:52.023162051 [I/WhiteNavigationLice]: Modify naviBarColor to 0xFFF0F0F0, and windowBackgroundColor is -328966   /windowIsTranslucent:false
2024-12-06T14:42:52.056058282 [I/SurfaceFactory]: [static] sSurfaceFactory = com.transsion.rt.SurfaceFactoryImpl@f2a0984
2024-12-06T14:42:52.057326974 [I/MsyncFactory]: [static] sMsyncFactory = com.mediatek.view.impl.MsyncFactoryImpl@3e4756d
2024-12-06T14:42:52.060738358 [W/libc]: Access denied finding property "ro.vendor.scaler_version"
2024-12-06T14:42:52.061366051 [W/libc]: Access denied finding property "ro.vendor.scaler_version"
2024-12-06T14:42:52.079497743 [I/PowerHalWrapper]: PowerHalWrapper.getInstance 
2024-12-06T14:42:52.186183512 [I/BufferQueueConsumer]: [](id:9a800000000,api:0,p:-1,c:2472) connect: controlledByApp=false
2024-12-06T14:42:52.187619205 [I/BLASTBufferQueue]: [ViewRootImpl[MainActivity]#0] constructor()
2024-12-06T14:42:52.196639282 [E/OpenGLRenderer]: Device claims wide gamut support, cannot find matching config, error = EGL_SUCCESS
2024-12-06T14:42:52.196841128 [W/OpenGLRenderer]: Failed to initialize 101010-2 format, error = EGL_SUCCESS
2024-12-06T14:42:52.209030051 [I/BufferQueueProducer]: [ViewRootImpl[MainActivity]#0(BLAST Consumer)0](id:9a800000000,api:1,p:2472,c:2472) connect: api=1 producerControlledByApp=true
2024-12-06T14:42:52.217322358 [I/mali]: [DebugProp] GPU Debug-Mechanism Disable !

2024-12-06T14:42:52.260828743 [E/OpenGLRenderer]: fbcNotifyFrameComplete error: undefined symbol: fbcNotifyFrameComplete
2024-12-06T14:42:52.260910974 [E/OpenGLRenderer]: fbcNotifyNoRender error: undefined symbol: fbcNotifyNoRender
2024-12-06T14:42:53.180681435 [I/BufferQueueConsumer]: [](id:9a800000001,api:0,p:-1,c:2472) connect: controlledByApp=false
2024-12-06T14:42:53.181121666 [I/BLASTBufferQueue]: [ViewRootImpl[MainActivity]#1] constructor()
2024-12-06T14:42:53.188924743 [I/BufferQueueProducer]: [ViewRootImpl[MainActivity]#1(BLAST Consumer)1](id:9a800000001,api:1,p:2472,c:2472) connect: api=1 producerControlledByApp=true
2024-12-06T14:42:54.555327897 [I/WhiteNavigationLice]: Modify naviBarColor to 0xFFF0F0F0, and windowBackgroundColor is -328966   /windowIsTranslucent:false
2024-12-06T14:42:54.616076205 [I/BufferQueueConsumer]: [](id:9a800000002,api:0,p:-1,c:2472) connect: controlledByApp=false
2024-12-06T14:42:54.616605743 [I/BLASTBufferQueue]: [ViewRootImpl[ApplicationLogsActivity]#2] constructor()
2024-12-06T14:42:54.621315589 [I/BufferQueueProducer]: [ViewRootImpl[ApplicationLogsActivity]#2(BLAST Consumer)2](id:9a800000002,api:1,p:2472,c:2472) connect: api=1 producerControlledByApp=true
```
Crash dump: [Crash 2024-12-06 14.42.31.dmp](https://github.com/user-attachments/files/18034527/Crash.2024-12-06.14.42.31.dmp)

**Comment by Zilko** [comment]
fixed in 1.2.1

**Comment by bikethatreportsbugs36** [comment]
It still happens to me though
